oop/homework.py

```diff
@@ -40,23 +40,15 @@ class Cat:
     * Implement get_average_speed and return average_speed
 
     """
-    """Add to class saturation_level variable with value 50"""
-    saturation_level = 50
 
     def __init__(self, age):
         self.age = age
         self.average_speed = self._set_average_speed()
         self.saturation_level = 50
 
     def eat(self, product):
-        if product == "fodder":
-            self._increase_saturation_level(10)
-        if product == "apple":
-            self._increase_saturation_level(5)
-        if product == "milk":
-            self._increase_saturation_level(2)
-        if product == "another_food":
-            self._increase_saturation_level(0)
+        food_inc = {"fodder": 10, "apple": 5, "milk": 2, "another_food": 0}
+        self._increase_saturation_level(food_inc.get(product))
```

Comment:
до речі
ще звернув увагу на одну штуку
якщо кіт з'їсть їжу, якої нема в словнику, то поверне TypeError
тому треба додати до dict.get() ще дефолтне значення
```suggestion
        self._increase_saturation_level(food_inc.get(product, default=0))
```

Reply:
https://stackoverflow.com/questions/24463202/typeerror-get-takes-no-keyword-arguments